lua/copilot/suggestion.lua

```diff
@@ -487,7 +487,9 @@ function mod.accept(modifier)
   vim.schedule_wrap(function()
     vim.api.nvim_feedkeys(vim.api.nvim_replace_termcodes("<Space><Left><Del>", true, false, true), "n", false)
       local bufnr = vim.api.nvim_get_current_buf()
-    local encoding = vim.api.nvim_get_option_value('fileencoding', { buf = bufnr })
+    local encoding = vim.api.nvim_get_option_value("fileencoding", { buf = bufnr }) ~= ""
+        and vim.api.nvim_get_option_value("fileencoding", { buf = bufnr })
+      or vim.api.nvim_get_option_value("encoding", { scope = "global" })
```

Comment:
I find this is easier to read IMO.
```suggestion
    local encoding = vim.api.nvim_get_option_value("fileencoding", { buf = bufnr })
    if encoding == "" then
      encoding = vim.api.nvim_get_option_value("fileencoding", { scope = "global" })
    end
```